src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -122,6 +122,10 @@ server::routes_t get_schema_registry_routes(ss::gate& gate, one_shot& es) {
       ss::httpd::schema_registry_json::get_schemas_ids_id_versions,
       wrap(gate, es, get_schemas_ids_id_versions)});
 
+    routes.routes.emplace_back(server::route_t{
+      ss::httpd::schema_registry_json::get_schemas_ids_id_subjects,
+      wrap(gate, es, get_schemas_ids_id_subjects)});
+
```

Comment:
Squash into previous commit because, without this change, the `get_schemas_ids_id_subjects` will fail to run.

Reply:
Good point. I modeled this stack after [one of Ben's](https://github.com/redpanda-data/redpanda/pull/3299) from a while ago, but maybe this level of granularity is no longer necessary. @BenPope any opinion on that?

Reply:
the most important factor in commit ordering is that each commit compiles so that bisecting is easier. we don't test or enforce this property, but when we see it it's a nice thing to flag.

ensuring tests pass with each commit is probably much harder. i'd say it's best effort. nice if doesn't hurt the ability to review the pr.

Reply:
Looks good. Closing out.